app/assets/stylesheets/content/_tables.sass

```diff
@@ -26,14 +26,59 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
+table
+  td
+    padding: 3px 6px
+  p
+    margin: 0
+
+  &.files tr.file td
+    text-align: center
+    &.filename
+      text-align: left
+      padding-left: 24px
+    &.digest
+      font-size: 80%
```

Comment:
Are you sure that those ones are still in use? Where do we display an attachments digest to the user?